ProGetAutomation/Functions/Add-ProGetAsset.ps1

```diff
@@ -0,0 +1,59 @@
+function Add-ProGetAsset

```

Comment:
Rename this to `Set-ProGetAsset` since it both creates and updates assets.

Reply:
Done, in the current case it over writes whatever was there. I could also change the rest call from a post to a put and that would make it only create new one if an asset doesnt exist. depends on what you want the function to do.

Reply:
per our conversation we discussed keeping it to write over current asset